src/lib/core/components/visualizations/implementations/MosaicVisualization.tsx

```diff
@@ -802,55 +812,192 @@ function MosaicViz(props: Props<Options>) {
 }
 
 function TwoByTwoStats(props?: {
-  pValue?: number | string;
-  oddsRatio?: number | string;
-  orInterval?: number | string;
-  relativeRisk?: number | string;
-  rrInterval?: number | string;
+  // reflecting 2x2 stats table content
+  chiSq?: twoByTwoStatsContent;
+  fisher?: twoByTwoStatsContent;
+  prevalence?: twoByTwoStatsContent;
+  oddsRatio?: twoByTwoStatsContent;
+  relativeRisk?: twoByTwoStatsContent;
+  sensitivity?: twoByTwoStatsContent;
+  specificity?: twoByTwoStatsContent;
+  posPredictiveValue?: twoByTwoStatsContent;
+  negPredictiveValue?: twoByTwoStatsContent;
+  facetVariableDetails?: facetVariableDetailsType;
 }) {
-  // Temporarily disabled---See https://github.com/VEuPathDB/web-eda/issues/463
-  if (1)
-    return (
-      <div
-        style={{
-          margin: '15px 0',
-          height: '2em',
-          width: '750px',
-        }}
-      >
-        <i>Stats table coming soon!</i>
-      </div>
-    );
-
   return props != null ? (
     <div
-      className="MosaicVisualization-StatsTable"
-      style={{ margin: '15px 0' }}
+      className="stats-table"
+      style={
+        props.facetVariableDetails != null
+          ? { width: '750px' }
+          : { margin: '15px 0', marginLeft: '0.9em', width: '750px' }
+      }
     >
       <table>
         {' '}
         <tbody>
           <tr>
-            <th></th>
-            <th className="numeric-header">Value</th>
-            <th className="numeric-header">95% confidence interval</th>
+            {/* <th></th> */}
+            <td className="stats-table_top-empty-cell"></td>
+            <td className="stats-table_top-empty-cell"></td>
+            <th
+              className="stats-table_top-left-cell"
+              style={{ background: MEDIUM_GRAY, textAlign: 'right' }}
+            >
+              Value
+            </th>
+            <th
+              className="stats-table_top-cell"
+              style={{ background: MEDIUM_GRAY, textAlign: 'center' }}
+            >
+              95% CI
+            </th>
+            <th
+              className="stats-table_top-right-cell"
+              style={{ background: MEDIUM_GRAY, textAlign: 'right' }}
+            >
+              P-value
+            </th>
           </tr>
           <tr>
-            <th>P-value</th>
-            <td className="numeric">
-              {props.pValue != null ? quantizePvalue(props.pValue) : 'N/A'}
+            <td className="stats-table_leftmost-cell">
+              Association between 2 categorical variables
+            </td>
+            <td className="stats-table_middle-cell">
+              <b>Chi-squared (df=1)</b>
+            </td>
+            <td>{props.chiSq?.value ?? 'n/a'}</td>
```

Comment:
Looks like we're preferring lowercase for `n/a` in the mockups included at the top of this PR. As a result, let's [update `ContTableStats` to use lowercase as well](https://github.com/VEuPathDB/web-eda/pull/1625/files#diff-556a996edbde2eb7605da318eb9e2d90be4af8ee28a245e9193524488a1960b8L888-L901).

Reply:
Thank you for your detailed explanations and suggestions @jernestmyers 👍 I did address your feedbacks so can you please check if they are what you had in mind? https://github.com/VEuPathDB/web-eda/pull/1625/commits/bc3462accb5d146ff18dc2d676e7ef78b8f95604